src/pages/iou/request/step/IOURequestStepParticipants.tsx

```diff
@@ -53,20 +60,24 @@ function IOURequestStepParticipants({
         return translate(TransactionUtils.getHeaderTitleTranslationKey(transaction));
     }, [iouType, transaction, translate, isSplitRequest]);
 
-    const receiptFilename = lodashGet(transaction, 'filename');
-    const receiptPath = lodashGet(transaction, 'receipt.source');
-    const receiptType = lodashGet(transaction, 'receipt.type');
-    const newIouType = useRef();
+    const receiptFilename = transaction?.filename;
+    const receiptPath = transaction?.receipt?.source;
+    const receiptType = transaction?.receipt?.type;
+    const newIouType = useRef<IouRef>();
 
     // When the component mounts, if there is a receipt, see if the image can be read from the disk. If not, redirect the user to the starting step of the flow.
     // This is because until the request is saved, the receipt file is only stored in the browsers memory as a blob:// and if the browser is refreshed, then
     // the image ceases to exist. The best way for the user to recover from this is to start over from the start of the request process.
     useEffect(() => {
-        IOU.navigateToStartStepIfScanFileCannotBeRead(receiptFilename, receiptPath, () => {}, iouRequestType, iouType, transactionID, reportID, receiptType);
+        IOU.navigateToStartStepIfScanFileCannotBeRead(receiptFilename ?? '', receiptPath ?? '', () => {}, iouRequestType, iouType, transactionID, reportID, receiptType ?? '');
     }, [receiptType, receiptPath, receiptFilename, iouRequestType, iouType, transactionID, reportID]);
 
     const updateRouteParams = useCallback(() => {
-        IOU.updateMoneyRequestTypeParams(navigation.getState().routes, newIouType.current);
+        const navigationState = navigation.getState();
+        if (!navigationState || !newIouType.current) {
```

Comment:
Make sure this update doesn't cause regression

Reply:
+1